ambry-clustermap/src/main/java/com/github/ambry/clustermap/AmbryPartition.java

```diff
@@ -137,12 +137,18 @@ public String getPartitionClass() {
     return partitionClass;
   }
 
+  @Override
+  public String getResourceName() {
+    return clusterManagerCallback.getResourceNameForPartition(this);
+  }
+
   @Override
   public JSONObject getSnapshot() {
     JSONObject snapshot = new JSONObject();
     snapshot.put(PARTITION_ID, toPathString());
     snapshot.put(PARTITION_WRITE_STATE, getPartitionState().name());
     snapshot.put(PARTITION_CLASS, getPartitionClass());
+    snapshot.put(PARTITION_RESOURCE_ID, getResourceName());
```

Comment:
Should we align the naming convention between PARTITION_RESOURCE_ID and resourceName?

Reply:
Sure, to make it consistent, I'll update it to PARTITION_RESOURCE_NAME.